pages/myPlants.js

```diff
@@ -0,0 +1,45 @@
+import Footer from "@/components/Footer";
+import PlantList from "@/components/PlantList";
+import styled from "styled-components";
+import useLocalStorageState from "use-local-storage-state";
+
+export default function MyPlantsPage() {
+  const [plants, setPlants] = useLocalStorageState("favourites");
+
+  const favouritePlants = plants
+    ? plants.filter((plant) => plant.isFavourite)
+    : [];
+
+  function toggleFavourite(id) {
+    setPlants((prevPlants) =>
+      prevPlants.map((plant) =>
+        plant.id === id ? { ...plant, isFavourite: !plant.isFavourite } : plant
+      )
+    );
+  }
+
+  return (
+    <div>
```

Comment:
**As mentioned above.**

- Consider replacing `<div>` with `<main>` here to improve the semantic structure. [The importance of the main element.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) 😁